### ISSUE TITLE
The latest version (v1.0.2) is not available in NuGet

### ISSUE THREAD

**Issue by leleuj** [Jasig/dotnet-cas-client#20]
The last version is the 1.0.2, which is a security fix.

But the last version available in NuGet is the 1.0.1: http://www.nuget.org/packages/DotNetCasClient. I fear that people are still using this unsafe one.

Can we update the latest version 1.0.2 in NuGet?

Thanks.

**Comment by rdev5** [comment]
Is this project still being actively maintained and is it stable for production? Would also like to know if there are any tests available.

Thanks!

**Comment by mmoayyed** [comment]
Yes it's perfectly active and stable. We just have not baked a whole lot of changes into it recently. what sort of tests are you looking for?

**Comment by scottt732** [comment]
I built and uploaded a package to NuGet  For some reason it is't doing the web.config transformation automatically anymore, though I think that the feature might have caused more configuration-related issues than it solved.  If anyone has a project that they can easily verify that it's working properly.  Also, I uploaded the symbols package to the symbol server, which should help users stepping through the debugger.
